paddle/fluid/pir/dialect/operator/ir/manual_api.cc

```diff
@@ -76,5 +88,23 @@ pir::OpResult embedding_grad(pir::Value x,
   }
 }
 
+pir::OpResult split_with_num_grad(std::vector<pir::Value> out_grad, int axis) {
+  auto out_grad_combine_op =
+      APIBuilder::Instance().GetBuilder()->Build<pir::CombineOp>(out_grad);
+  paddle::dialect::SplitGradOp split_grad_op =
+      APIBuilder::Instance().GetBuilder()->Build<paddle::dialect::SplitGradOp>(
+          out_grad_combine_op.out(), axis);
+  return split_grad_op.result(0);
+}
+
+pir::OpResult split_with_num_grad(std::vector<pir::Value> out_grad,
```

Comment:
```suggestion
pir::OpResult split_with_num_grad(std::vector<pir::Value>& out_grad,
```
虽然 pir::Value 的构造成本很低，但这里还是建议传入vector<>&，如有必要，也要加上const 限定符

Reply:
这个在vjp处处理直接调用了invoke的api, 如果没有其他直接调用api 的情况此处后续会删除